editor/src/components/canvas/canvas-strategies/canvas-strategies.ts

```diff
@@ -15,28 +15,44 @@ import {
 } from './canvas-strategy-types'
 import { flexAlignParentStrategy } from './flex-align-parent-strategy'
 import { flexGapStrategy } from './flex-gap-strategy'
+import { resizeAbsoluteStrategy } from './resize-absolute-strategy'
 
-const RegisteredCanvasStrategies: Array<CanvasStrategy> = [flexGapStrategy, flexAlignParentStrategy]
+const RegisteredCanvasStrategies: Array<CanvasStrategy> = [
+  resizeAbsoluteStrategy,
+  flexGapStrategy,
+  flexAlignParentStrategy,
+]
 
 export function pickDefaultCanvasStrategy(
   editorState: EditorState,
   sessionProps: SelectModeCanvasSessionProps,
   sessionState: SelectModeCanvasSessionState,
 ): CanvasStrategy | null {
-  const applicableStrategies = RegisteredCanvasStrategies.map((s) => ({
-    fitness: s.fitnessFn(editorState, sessionProps, sessionState),
-    strategy: s,
-  })).filter((s) => {
-    // discard strategies with 0 fitness
-    return s.fitness > 0
```

Comment:
I think master was already doing the same (only calculating fitness once), the only difference is that it discards strategies with 0 fitness

Reply:
I implemented what we discussed, which was to have `null` indicate that it cannot be handled by this strategy at all.